evm/evm.go

```diff
@@ -0,0 +1,70 @@
+package evm
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/consensus"
+	"github.com/ethereum/go-ethereum/consensus/beacon"
+	"github.com/ethereum/go-ethereum/core"
+	gethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/core/vm"
+	"github.com/ethereum/go-ethereum/params"
+	"github.com/polymerdao/monomer"
+	"github.com/polymerdao/monomer/utils"
+)
+
+var (
+	// MonomerGenesisRootHash is the known root hash of the monomer ethereum state trie after all predeployed contracts are created.
+	MonomerGenesisRootHash = common.HexToHash("0x4e171626bc7f95d0b053dab23c87327ee0266dda88b213a3e1f1357b741c0c35")
+	// MonomerEVMTxOriginAddress is the address used for executing transactions in the monomer EVM.
+	MonomerEVMTxOriginAddress = common.HexToAddress("0xae76d9126c32d7eafe3f5b7bd5a7b44f2d5bb8b1")
```

Comment:
Referencing discussion [here](https://github.com/polymerdao/monomer/pull/84#discussion_r1697503595):

For deterministic addresses that we have control over, it may be better to go with something that screams "decided on by a person", instead of raising the questions like
- "how was this value chosen"
- "where was this value generated / sourced from"
- "is this value potentially out of date"

For example, the OP predeploys are addressed `0x4200000000000000000000n`. (The most juvenile namespace :) )